e2e/pages/organizations/developers/developers.spec.js

```diff
@@ -2,22 +2,49 @@ import DevelopersPage from './developers.po';
 import Hooks from '../../../utilities/hooks';
 import ContactComponent from '../../../components/contact/contact.po';
 import LoginComponent from '../../../components/login/login.po';
-
-let contact, hooks, login, page;
+import ActionBarComponent from '../../../components/action-bar/action-bar.po';
+import ToastComponent from '../../../components/toast/toast.po';
+import AddressComponent from '../../../components/address/address.po';
+let actionBar, address, contact, hooks, login, page, toast;
 
 describe('the Developers page', () => {
+  let timestamp = (new Date()).getTime();
+  let website = 'https://website' + timestamp + '.com';
+  let poc = {
```

Comment:
Using full names is helpful for readability:
```suggestion
  let developerContact = {
```